src/main/java/co/zecko/retailer/client/httpClient/ProductClient.java

```diff
@@ -78,4 +78,22 @@ public ProductData findById(String id, String imagesBefore, String imagesAfter,
 
         return get(url, queryParams, headers, new TypeReference<>() {});
     }
+
+    public ProductData findRealTimeData(String id)
+        throws ZeckoException, IOException, InterruptedException {
+        Map<String, String> headers = getBaseHeaders(zeckoAccessToken);
+        Map<String, String> queryParams = new HashMap<>();
+        String routeUri = "";
+
+        if (StringUtils.isEmpty(id)) {
+            String message = "Missing required parameter: id";
+            throw new ZeckoException(message, HttpStatus.BAD_REQUEST);
+        }
+
+        routeUri = String.format("/realtime/%s", id);
```

Comment:
```suggestion
        String routeUri = String.format("/realtime/%s", id);
```

Declaration on line 86 can be removed